app/gemini-bu/app.toml

```diff
@@ -137,12 +137,20 @@ task-slots = ["sys"]
 
 [tasks.hiffy]
 name = "task-hiffy"
-features = ["h753", "stm32h7", "itm", "i2c", "gpio", "spi", "qspi", "hash"]
-priority = 5
+features = ["h753", "stm32h7", "itm", "i2c", "gpio", "spi", "qspi", "hash", "sprot"]
+priority = 7
 max-sizes = {flash = 32768, ram = 16384 }
 stacksize = 2048
 start = true
-task-slots = ["sys", "i2c_driver", "hf", "hash_driver"]
+task-slots = ["sys", "i2c_driver", "hf", "hash_driver", "sprot"]
+
+[tasks.sprot]
+name = "drv-stm32h7-sprot-server"
+priority = 5
+max-sizes = {flash = 16384, ram = 32768}
+stacksize = 16384
```

Comment:
The stack size on the SP was inflated. I can take it down to 4K based on running the tests and what humility stackmargin says.
On the RoT, it is pulling Sprockets into the task. That plus other un-identified things push the stack just over 8K.
I expect it will improve over time.